test/billing_test.go

```diff
@@ -0,0 +1,16 @@
+package test
+
+func (s *CLITestSuite) TestCostList() {
+	tests := []CLITest{
+		{args: "billing cost list --start-date 2023-01-01 --end-date 2023-01-02", fixture: "billing/list.golden"},
+		{args: "billing cost list --start-date 2023-01-01 --end-date 2023-01-02 -o json", fixture: "billing/list_json.golden"},
```

Comment:
Match the style of our other tests!
```suggestion
		{args: "billing cost list --start-date 2023-01-01 --end-date 2023-01-02 -o json", fixture: "billing/cost/list-json.golden"},
```